packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -24,6 +28,7 @@ const ProgressBar = React.forwardRef<HTMLDivElement, ProgressBarProps>(
       size = "small",
       indicatorColor = "colorBackgroundPrimaryWeak",
       backgroundColor = "colorBackgroundWeak",
+      indicatorStyle,
```

Comment:
Why not just create a token `colorBackgroundIndicator` for example, replace the default token on line 29, and override it on the talent app?

Reply:
We could do but I don't believe that `colorBackgroundIndicator` is a good variable according with many directives

 **"Design tokens are name and value pairings that represent small, repeatable design decisions."**

This is not a decision, this is a more flexible and customizable component

Reply:
Yes, I agree. But this is happening just because the MVP of the theming project wants to change only small stuff. The DS should not be affected by this decision. And the maintenance of the talent app would be strongly affected by it. 

Honestly, I am not sure which one is the worse option. But we need to figure out a better way to do it.